### PR TITLE
WebUI: certmap match

### PR DESCRIPTION
**Pull request by pvomacka**
WebUI: add support for certmap match command. 

PR contains also certmap rule patches from pullrequest #400 (I will rebase once #400 will be merged) because they are necessary. It also requires PRs #398 and #516.

https://pagure.io/freeipa/issue/6601

### CODE REVIEW

**Comment by pvomacka** [comment]
Rebased. PR #400 already merged.

**Comment by pvomacka** [comment]
In last sync I changed string of clear button title.

File: install/ui/src/freeipa/field.js [hunk 1]
```diff
@@ -819,6 +819,15 @@ field.Adapter = declare(null, {
     result_index: 0,
 
     /**
+     * When result of API call is an array of object this object index
+     * allows to specify exact object in array according to its position.
+     * Default value is null which means do not use object_index.
+     *
+     * @type {Number|null}
+     */
+     object_index: null,
```
Comment (pvoborni) [inline install/ui/src/freeipa/field.js]: incorrect leading space

Comment (pvomacka) [inline install/ui/src/freeipa/field.js]: Fixed.

**Comment by pvomacka** [comment]
@pvoborni  Yes, we should make a lint rule for leading spaces.

**Comment by pvoborni** [comment]
But let's wait with pushing for travis, to be sure.

**Comment by dkupka** [comment]
master:

* 6be32edde0ae16473d4d109747adae78f9d725e4 WebUI: Add possibility to turn of autoload when details.load is called
* 1d6cc35c03669ea67d9e9ee9ca0ff62401d1b157 WebUI: Possibility to choose object when API call returns list of objects
* 358caa7da44c997b505f54ec70cb6be58d188751 WebUI: Add Adapter for certmap_match result table
* 61cd4372e142662c06c881886709fe1b573102a9 WebUI: Add cermapmatch module

**Pull request closed without merging**